src/common/component/Input/Input.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { InputHTMLAttributes } from 'react';
+
+import {
+  inputContainerStyle,
+  inputStyle,
+  inputWarpperStyle,
+  sizeStyle,
+  variantStyle,
+} from '@/common/component/Input/Input.style';
+import { InputHeight, InputVariant } from '@/common/type/design';
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  variant: InputVariant;
+  height?: InputHeight; //default: medium(4.8rem)
```

Comment:
보운님이 의도하신 바가 맞다면, `size`라는 네이밍이 더욱 잘 어울릴 것 같습니다.

Reply:
height만 바뀌어서 height라고 했는데, size가 더 직관적인 네이밍인 것 같네요 ! 그럼 InputHeight 타입의 short, medium의 네이밍도 small, medium으로 하는 게 좋을까요~?

Reply:
아 그리고 size로 네이밍하지 않은 이유가 InputHTMLAttributes를 extends하면 내재된 size의 속성이 number이기 때문에 string인 타입으로 지정이 불가하더라구요 ! 
이를  아래 처럼 inputSize로 네이밍하는 건 어떤가요?
```suggestion
  inputSize?: InputSize; //default: medium(p: 1.2rem)
```

Reply:
`Input` 컴포넌트의 `size` 프롭의 네이밍을 `inputSize`라고 하면 어색할 것 같아요 !
보운님께서 말씀하신 상속 과정에서의 타입 호환성을 해결하기 위해 `Omit` 유틸리티 타입을 사용해요!

```ts
interface InputProps extends Omit<HTMLAttributes<HTMLInputElement>, 'size'> {}
```
처럼 적어준다면 기존의 HTMLInputElement 타입이 가지고 있던 `size`라는 타입을 제외한 타입을 생성할 수 있습니다 !!

Reply:
아 이런상황을 위한 유틸리티 타입이 있었군요 ! 
좋은 정보 감사합니다 😺 수정완료했습니다 ~!